test/test_emulate_sample.jl

```diff
@@ -1,34 +0,0 @@
-import JLD2
-import Statistics: mean
```

Comment:
why is this test deleted?

Reply:
I thought that `emulate_sample` is a bit misleading if I add a test for the `calibrate` function. However, that test should really go in the model interface tests, and this can stay `emulate_sample`!